data_processors/pipeline/lambdas/orchestrator.py

```diff
@@ -223,9 +217,11 @@ def next_step(this_workflow: Workflow, skip: dict, context=None):
 
         return results
 
-    elif this_workflow.type_name.lower() == WorkflowType.DRAGEN_WGS_QC.value.lower() and \
-            this_workflow.end_status.lower() == WorkflowStatus.SUCCEEDED.value.lower():
-        logger.info(f"Received DRAGEN_WGS_QC workflow notification")
+    elif (
+            this_workflow.type_name.lower() == WorkflowType.DRAGEN_WGS_QC.value.lower() or
+            this_workflow.type_name.lower() == WorkflowType.DRAGEN_WTS_QC.value.lower()
+    ) and this_workflow.end_status.lower() == WorkflowStatus.SUCCEEDED.value.lower():
```

Comment:
I still wish to keep this just WGS check here then triggering T/N. More in below comment.

Reply:
Partially rolled-back in 1186883cb382396e469671b2dc8b2c768c39d448